packages/fluentui/react-northstar/src/utils/whatInput.ts

```diff
@@ -106,8 +106,10 @@ const addListeners = (eventTarget: Window) => {
   eventTarget.addEventListener('keyup', eventBuffer, true);
 };
 
+type UIEventOnElement<T> = UIEvent & { currentTarget: T }
```

Comment:
#20024 Adds correct typings to the event listeners, so this manual typing is no longer required